mne/decoding/tests/test_transformer.py

```diff
@@ -237,16 +237,16 @@ def test_temporal_filterer():
 
     # Add tests for different combinations of l_freq and h_freq
     filt = TemporalFilter(l_freq=5, h_freq=15, sfreq=100,
-                          filter_length='auto', l_trans_bandwidth='auto',
-                          h_trans_bandwidth='auto')
+                          filter_length='auto', l_trans_bandwidth=None,
+                          h_trans_bandwidth=None)
```

Comment:
Yes, `auto` should work. [Link](https://github.com/mne-tools/mne-python/blob/master/mne/filter.py#L1872)